frontend/src/metabase-lib/lib/metadata/Metadata.ts

```diff
@@ -105,8 +105,8 @@ export default class Metadata extends Base {
     return (fieldId != null && this.fields[fieldId]) || null;
   }
 
-  question(card) {
-    return new Question(card, this);
+  question(cardId) {
```

Comment:
This method was only ever used in one place, so I'm stealing it since we need it for grabbing questions out of cache.